apps/docs/content/components/forms/TextArea.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: TextArea
-description: A TextArea serves as a multi-line, plain-text editing interface, ideal for accommodating substantial amounts of free-form text input from users.
+description: A textarea serves as a multi-line, plain-text editing interface.
```

Comment:
I have the same comment as I had for ListBox. The capital A suggests that area is a separate word.